config.go

```diff
@@ -1,46 +1,61 @@
 package main
 
 import (
-	"github.com/devangel/config"
 	"os"
 	"path/filepath"
-	"strings"
-	//"fmt"
+
+	"github.com/devangel/config"
 )
 
 var Config = config.NewConfig("force")
 
-func GetSourceDir() (src string, err error) {
-	// Last element is default
-	var sourceDirs = []string{
-		"src",
-		"metadata",
+var sourceDirs = []string{
+	"src",
+	"metadata",
+}
+
+// IsSourceDir returns a boolean indicating that dir is actually a Salesforce
+// source directory.
+func IsSourceDir(dir string) bool {
```

Comment:
Just like the previous code, this gives precedence for the `src` directory before the `metadata` directory. Unlike the previous code, it doesn't look too far up the parent tree to determine if it has found it.

As for the `Package` folder, what is the semantic for that? Are you saying that we should also match `Package/package.xml`? Or ignore it? Because this code should ignore it, although it won't ignore `Package/src/package.xml` or `Package/metadata/package.xml`.